src/NGen.cpp

```diff
@@ -3,6 +3,8 @@
 #include <string>
 #include <unordered_map>
 
+#include <utilities/span.hpp>
```

Comment:
Noted 🤨 but sure if we want to make sure it compiles for now.

Reply:
To memorialize our Slack conversation - that was the intention.